jira/client.py

```diff
@@ -3322,24 +3323,24 @@ def backup_download(self, filename=None):
             logging.error(ioe)
         return None
 
-    def current_user(self):
-        """Returns the username of the current user.
+    def current_accountid(self):
```

Comment:
Keep function name and *if* possible make it return same resul with both old/new servers. If not possible change its description and document that on newer instances it will receive the accountid.

If both return types can be returned there are two approaches: either to add extra function (current_accountid) or to add optional parameter that mentions that you want accountid instead of username to be returned.

Later if we are forced we can change the meaning of the default parameter.

The main idea is that if someone that code that uses the newer library (or newer jira server) should not break. Is not always possible but we should do out best.

Reply:
method name is current_user not currect_username which means that "user" can be anything, either username, userid, a disctionary that describes the user, an user object.

If the name of the function would have being different it would be total different story.

Reply:
You're right in this case, but I don't think it's possible for the rest of the changes.